src/context.ts

```diff
@@ -190,6 +197,43 @@ ${code.join('\n')}
     };
   }
 
+  async waitForTimeout(time: number) {
+    if (this._currentTab && !this._javaScriptBlocked())
```

Comment:
Why not always `await new Promise(f => setTimeout(f, time));` ?

Reply:
To make align with the browser clock.